README.md

```diff
@@ -46,7 +69,77 @@ npm install
 
 ## Usage
 
-TBD
+This client assumes:
+
+* a remote Wallet Attached Storage server is available
+* the app is able to create and store a public-private key pair, see the
+  [App Identity and Key Management](#app-identity-and-key-management) section
+  for more details.
+
+Example key pair generation (for an in-browser SPA), for persistence in local
+storage.
+
+```ts
+import { Ed25519Signer } from '@did.coop/did-key-ed25519'
+
+const appDidSigner = await Ed25519Signer.generate()
+```
+
+Example storing and loading:
+
+```ts
+// Serialize to JSON for storage
+const exportedKeyPair = appDidSigner.toJSON()
+localStorage.setItem('app-DID', JSON.stringify(exportedKeyPair))
+
+// Load from storage and turn back into a DID Signer
+const loadedKeyPair = localStorage.getItem('app-DID')
+const appDidSigner = Ed25519Signer.fromJSON(loadedKeyPair)
+```
+
+Create a Wallet Attached Storage Client:
+
+```ts
+import { WasClient } from '@did-coop/wallet-attached-storage'
```

Comment:
```suggestion
import { WasClient } from '@did-coop/wallet-attached-storage'
```

Can we export a more explicit name like `WalletAttachedStorageClient` or just `WalletStorageClient`?
IMO it makes it all more intelligble and easy to grok if we avoid 'Was', which I think folks won't immediately recognize the meaning of.

Reply:
Good point, will change